From: Jon Bernard <jbernard@debian.org>
Date: Sun, 27 Mar 2011 21:34:20 -0400
Subject: Fix spelling mistake in cgcreate
To: libcg-devel@lists.sourceforge.net

---
 src/tools/cgcreate.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/tools/cgcreate.c b/src/tools/cgcreate.c
index a2a21b2..29ef27e 100644
--- a/src/tools/cgcreate.c
+++ b/src/tools/cgcreate.c
@@ -53,7 +53,7 @@ static void usage(int status, const char *program_name)
 		fprintf(stdout, "  -f, --fperm mode		Group "\
 			"file permissions\n");
 		fprintf(stdout, "  -d, --dperm mode		Group "\
-			"direrory permissions\n");
+			"directory permissions\n");
 	}
 }
 
-- 
