Index: b/scripts/init.d/cgconfig.in
===================================================================
--- a/scripts/init.d/cgconfig.in
+++ b/scripts/init.d/cgconfig.in
@@ -21,10 +21,10 @@
 
 ### BEGIN INIT INFO
 # Provides:             cgconfig
-# Required-Start:
-# Required-Stop:
-# Should-Start:
-# Should-Stop:
+# Required-Start:       $remote_fs $syslog
+# Required-Stop:        $remote_fs $syslog
+# Default-Start:        2 3 4 5
+# Default-Stop:         0 1 6
 # Short-Description:    start and stop the WLM configuration
 # Description:          This script allows us to create a default configuration
 ### END INIT INFO
@@ -72,7 +72,7 @@ umount_fs() {
 
 start() {
         echo -n "Starting cgconfig service: "
-	if [ -f /var/lock/subsys/$servicename ]
+	if [ -f /var/lock/$servicename ]
         then
             log_warning_msg "lock file already exists"
             return
@@ -138,11 +138,11 @@ start() {
             done
         done
 	
-        touch /var/lock/subsys/$servicename
+        touch /var/lock/$servicename
         retval=$?
         if [ $retval -ne 0 ]
         then
-            log_failure_msg "Failed to touch " /var/lock/subsys/$servicename
+            log_failure_msg "Failed to touch " /var/lock/$servicename
             return $retval
         fi
         log_success_msg
@@ -182,7 +182,7 @@ stop() {
     echo -n "Stopping cgconfig service: "
     move_all_to_init_class
     umount_fs
-    rm -f /var/lock/subsys/$servicename
+    rm -f /var/lock/$servicename
     log_success_msg
 }
 
@@ -219,7 +219,7 @@ case $1 in
         common
         start;
         ;;
-    'restart')
+    'restart'|'force-reload')
         common
 	stop
         start
@@ -230,13 +230,13 @@ case $1 in
         start
         ;;
     'condrestart')
-        if [ -f /var/lock/subsys/$servicename ] ; then
+        if [ -f /var/lock/$servicename ] ; then
             stop
             start
         fi
         ;;
     'status')
-        if [ -f /var/lock/subsys/$servicename ] ; then
+        if [ -f /var/lock/$servicename ] ; then
             echo "Running"
         else
             echo "Stopped"
