Index: libcgroup/dist/Makefile.in
===================================================================
--- libcgroup.orig/dist/Makefile.in	2009-08-12 23:44:08.281152905 -0400
+++ libcgroup/dist/Makefile.in	2009-08-12 23:44:14.821149775 -0400
@@ -40,7 +40,7 @@
 	$(ACLOCAL_M4)
 mkinstalldirs = $(install_sh) -d
 CONFIG_HEADER = $(top_builddir)/config.h
-CONFIG_CLEAN_FILES = libcgroup.spec
+CONFIG_CLEAN_FILES =
 SOURCES =
 DIST_SOURCES =
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
